src/main/java/racing/Position.java

```diff
@@ -2,19 +2,24 @@
 
 public class Position {
     private static final int START_POSITION = 0;
+    private static final String DASH = "-";
 
     private int position;
 
     public Position() {
         this.position = START_POSITION;
     }
 
-    public Position(int position) {
-        this.position = position;
+    public void addOne() {
+        position++;
     }
 
-    public void add(int addend) {
-        position += addend;
+    public String dashOfPosition() {
+        StringBuilder stringBuilder = new StringBuilder();
+        for (int i = 0; i < position; i++) {
+            stringBuilder.append(DASH);
+        }
+        return stringBuilder.toString();
```

Comment:
String 클래스의 repeat 메서드를 활용해 보셔도 좋을 것 같아요
```suggestion
    public String dashOfPosition() {
        return DASH.repeat(position);
```